waterloop-site/src/theme/global.scss

```diff
@@ -697,10 +697,14 @@ h1 {
     font-family: IBM Plex Sans;
     font-style: normal;
     font-weight: 600;
-    font-size: 36px;
+    font-size: 1.5rem;
```

Comment:
The text gets quite small on wider screens like 4k monitors. I'd recommend using vw units instead of rem units (I found that 3vw works alright in your case), or setting the root font size to vw so that other fonts can inherit the dynamic sizing using rem units. See https://www.smashingmagazine.com/2016/05/fluid-typography/ for more details. 

cc: @jxrin @lynn-kim

Reply:
I had previously used 1.5rem for the SocialMediaWrapper and RoutingWrapper components as well, should I change the font size there too? @jmawloo

Reply:
I'll ask Jerin/Lynn if they think it's better to change it per component or just for the root font.